vmdb/app/models/provider.rb

```diff
@@ -0,0 +1,18 @@
+class Provider < ActiveRecord::Base
+  include AuthenticationMixin
+
+  belongs_to :zone
+
+  def verify_ssl=(val)
+    val = case val
+          when true then OpenSSL::SSL::VERIFY_PEER
+          when false then OpenSSL::SSL::VERIFY_NONE
+          else val
+          end
+    write_attribute(:verify_ssl, val)
```

Comment:
Should we make the column boolean, or is the goal to store the specific value to allow for other choices?